source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -296,6 +297,9 @@ def getContainerCpuMetricItems(metricJSON, hostName, cpuMetricNameToCollect, met
                     if (!@clusterMultilineEnabled.nil? && !@clusterMultilineEnabled.empty?)
                       telemetryProps["multilineEnabled"] = @clusterMultilineEnabled
                     end
+                    if (!@clusterKubernetesMetadataEnabled.nil? && !@clusterKubernetesMetadataEnabled.empty?)
+                      telemetryProps["metadataEnabled"] = @clusterKubernetesMetadataEnabled
```

Comment:
what are the following changes regarding this CAdvisorMetrics telemetry?

Reply:
i am not sure I understand

Reply:
I mean are there any changes needed in other places for this to collect this telemetry? Do we also need to collect telemetry such as labels and annotations when we added those later?

Reply:
we can add those later. this is just to report whether filter is enabled or not